### PR TITLE
Add CC-BY-2.5-AU license

### LINKED ISSUE #1171: New license request: CC-BY-2.5-AU
I tried submitting this in the web form. Got an error (attached at the end) but couldn't find where to report that error even. So I decided to skip that process altogether and just commit straight to GitHub.

**License's Full Name:** Creative Commons Attribution 2.5 Australia (CC BY 2.5 AU) 
**Short Identifier:** CC-BY-2.5-AU
**Source / URL:** https://creativecommons.org/licenses/by/2.5/au/
**Who is the license author or license steward?:** Creative Commons
**Example project URLs:** https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow
**Comments:** Found it being used in the git flow documentation by Atlassian. I noticed other country-specific-variants were already included in SPDX, but not this particular variant.

**SPDX Online Tool - License request - Error:**
```
Unexpected error, please email the SPDX technical workgroup that the following error has occurred:
Traceback (most recent call last):
File "/spdx/src/app/views.py", line 127, in submitNewLicense
  matchingIds, matchingType = utils.check_spdx_license(licenseText)
File "/spdx/src/app/utils.py", line 533, in check_spdx_license
  listedLicense = getListedLicense(licenseID)
File "/spdx/src/spdx-license-matcher/spdx_license_matcher/utils.py", line 88, in getListedLicense
  listed_license = licenseinfofactoryclass.getListedLicenseById(licenseId)
File "/usr/local/lib/python2.7/site-packages/jpype/_jpackage.py", line 62, in __call__
  raise TypeError("Package {0} is not Callable".format(self.__name))
TypeError: Package org.spdx.rdfparser.license.LicenseInfoFactory.getListedLicenseById is not Callable 
```

### PR DESCRIPTION
**Pull request by nicorikken**
Adds the Creative Commmons Attribution 2.5 Australia (CC BY 2.5 AU) license.

Closes #1171

Signed-off-by: Nico Rikken <nico.rikken@alliander.com>

### CODE REVIEW

**Comment by nicorikken** [comment]
I see #1265 so I'll include the optional tags as well in an additional commit.

File: src/CC-BY-2.5-AU.xml [hunk 1]
```diff
@@ -0,0 +1,403 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="CC-BY-2.5-AU"
+            name="Creative Commons Attribution 2.5 Australia">
```
Comment (seabass-labrax) [inline src/CC-BY-2.5-AU.xml:4]:
```suggestion
            name="Creative Commons Attribution 2.5 Australia" listVersionAdded="3.14">
```

Comment (seabass-labrax) [reply src/CC-BY-2.5-AU.xml:4]: This attribute is a relatively recent addition to the XML schema for the License List, so some of the older license XML submissions don't contain it. That being said, it's recommended for new submissions.

**Comment by seabass-labrax** [comment]
Hello @nicorikken! Thank you for taking the time to create the text files and XML for this pull request. I've seen you around and about on the REUSE mailing list, so it's great to see you here as well :)

**Comment by seabass-labrax** [comment]
I've run the tests locally; there were no errors detected.

**Review by seabass-labrax — APPROVED** [review]
Looks good to me :)

**Comment by jlovejoy** [comment]
LGTM. Updated assignee to give credit where credit due - thanks @nicorikken 
re-running tests, then will merge

**Comment by swinslow** [comment]
@seabass-labrax will add the listVersionAdded tag in a follow-up PR

**Comment by nicorikken** [comment]
Thanks!

**Pull request merged**